src/query/string_opfunc.c

```diff
@@ -4661,12 +5121,6 @@ db_string_regexp_replace (DB_VALUE * result, DB_VALUE * args[], int const num_ar
     if (cubregex::check_should_recompile (rx_compiled_regex, rx_compiled_pattern, pattern_string, reg_flags) == true)
       {
 	cubregex::clear (rx_compiled_regex, rx_compiled_pattern);
-	int pattern_length = pattern_string.size ();
```

Comment:
why is this code removed?

Reply:
these blocks are omitted in fixing conflict. thank you for checking it.